iron-overlay-manager.html

```diff
@@ -30,6 +30,31 @@
       element.style.zIndex = z;
     },
 
+    /**
+     * Returns the first opened overlay that has a backdrop.
+     */
+    _overlayWithBackdrop: function () {
+      for (var i = 0; i < this._overlays.length; i++) {
+        if (this._overlays[i].opened && this._overlays[i].withBackdrop) {
+          return this._overlays[i];
+        }
+      }
+    },
+
+    /**
+     * Calculates the minimum z-index for the overlay.
+     */
+    _getOverlayZ: function (overlay) {
+      var z = this._minimumZ;
+      if (overlay) {
+        var z1 = window.getComputedStyle(overlay).zIndex;
+        if (!isNaN(z1)) {
```

Comment:
There are some [potentially undesired coercion effects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#Confusing_special-case_behavior) when using `isNaN`. They may not be an issue here (since the string is typically coming from a style, not from user input), but worth keeping in mind.